gateway/server.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Alex Ellis 2017. All rights reserved.
```

Comment:
Why have you removed my personal copyright from this file?

Reply:
I only changed it to the broader "OpenFaaS Project" to match other files as I thought there weren't supposed to be individual names in the copyright. I will add your name back, in addition to the overarching "OpenFaaS Author(s)" line